datasets/cu/inventario_legislatura/crawler.py

```diff
@@ -61,5 +67,15 @@ def crawl(context: Context):
         entity.add("nationality", "cu")
         entity.add("topics", "role.pep")
 
+        occupancy = h.make_occupancy(
+            context,
+            entity,
+            position,
+            start_date=inception_date,
+            end_date=dissolution_date,
```

Comment:
make_occupancy will change these from current to ended in 2028

but @pudo can we assume this dataset is kept up to date and people who are no longer members are removed from the list? At least if they add an "ended" column we should get a warning from the audit.

Reply:
No I think the page we're crawling is super specific to this congress. The reason for crawling it is also a bit diplomatic (they're friends). Maybe after that we should have a script for the parliament's web site?